control-plane/csi-driver/src/bin/node/nodeplugin_nvme.rs

```diff
@@ -23,6 +41,19 @@ impl NvmeOperations for NvmeOperationsSvc {
             Some(map_wrapper) => map_wrapper.map,
             None => HashMap::new(),
         };
+        {
+            let mut cache = get_cache()?
+                .lock()
+                .map_err(|_| DeviceError::new("Could not access cache"))?;
```

Comment:
It's probably fine to unwrap here on mutex poisoning.

Reply:
Okay. I have used unwrap